.prettierignore

```diff
@@ -1 +1,3 @@
 .next
+package-lock.json
```

Comment:
The `package-lock.json` should be deleted as `package.json` is configured to use pnpm as the package manager.

```suggestion
```